### PR TITLE
chore(docs): Fix dark mode code examples

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes
Fix broken tab container for dark mode code examples:

* default dark mode JS code example wasn't showing
* custom dark mode examples were not in tab panels

| Before | After |
| --- | --- |
| Screenshot 2023-12-12 at 12 54 54 PM | Screenshot 2023-12-12 at 12 55 02 PM |
 | Screenshot 2023-12-12 at 12 44 51 PM | Screenshot 2023-12-12 at 12 50 49 PM |

#### Description of how you validated changes

``` yarn docs dev```

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/theming/dark-mode/react.mdx [hunk 1]
```diff
@@ -119,75 +119,79 @@ _Note: to see dark mode applied, change your OS preferences_
...
-          <ToggleButton value="dark">Dark</ToggleButton>
-          <ToggleButton value="system">System</ToggleButton>
-        </ToggleButtonGroup>
-        <Text>Current color mode: {colorMode}</Text>
-      </Card>
-    </ThemeProvider>
-  );
-};
-```
+          ```jsx
```
Comment (calebpollman) [inline docs/src/pages/[platform]/theming/dark-mode/react.mdx:131]: Are the changes to the example codeblock here just formatting related?

File: docs/src/pages/[platform]/theming/dark-mode/react.mdx [hunk 2]
```diff
@@ -38,9 +38,9 @@ Amplify UI comes with a default dark mode that you can use.
 
 _Note: the Amplify UI theme and any overrides like dark mode are scoped to the ThemeProvider. Changing the color mode in the example will only affect the example code._
 
-<Tabs.Container defaultValue="Javascript">
+<Tabs.Container defaultValue="JavaScript">
   <Tabs.List>
-    <Tabs.Item value="Javascript">Javascript</Tabs.Item>
+    <Tabs.Item value="JavaScript">JavaScript</Tabs.Item>
```
Comment (calebpollman) [inline docs/src/pages/[platform]/theming/dark-mode/react.mdx:43]: Nice!

**Review by calebpollman — COMMENTED** [review]
LGTM, just left a question abt formatting

File: docs/src/pages/[platform]/theming/dark-mode/react.mdx (hunk 1 above)
Comment (ioanabrooks) [reply docs/src/pages/[platform]/theming/dark-mode/react.mdx:131]: yes just spaces!

**Review by calebpollman — APPROVED** [review]

**Review by hbuchel — APPROVED** [review]

**Pull request merged**